gp-includes/things/translation.php

```diff
@@ -299,7 +299,7 @@ public function set_fields( $db_object ) {
 	}
 
 	public function for_export( $project, $translation_set, $filters =  null ) {
```

Comment:
Makes sense I'll update it later.

I think there's a problem though with my solution so it may need an update, I'll verify it before I merge.